invalidprocess/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"syscall"
+	"unicode/utf8"
+
+	"github.com/stackrox/rox/central/globaldb"
+	"github.com/stackrox/rox/central/processindicator/store/rocksdb"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/sac"
+)
+
+var log = logging.LoggerForModule()
+
+func main() {
+	shouldDelete := os.Getenv("DELETE_PROCESSES") == "true"
+
+	ctx := sac.WithAllAccess(context.Background())
+
+	store := rocksdb.New(globaldb.GetRocksDB())
```

Comment:
Is it a pre-requisite for some postgres migrations to clean the processInfo data in rocksDB ?
Is there a chance that process indicators in postgres could be impacted too ?

Reply:
This is not going to be shipped